### PR TITLE
Improved MenuButton Description

### PR DESCRIPTION
**Pull request by Maciej-Gorczyca**
I expanded description of MenueButton to explain how to create the items inside of the pop up as this is initially a mystery to many new people.

### CODE REVIEW

**Comment by Feniks-Gaming** [comment]
@Calinou this is minor change of 2 sentences, any chance I could get a quick review for it please? I know you guys are busy but it would be nice to push those easy wins with before 3.2 stable is out.

File: doc/classes/MenuButton.xml [hunk 1]
```diff
@@ -4,7 +4,8 @@
 		Special button that brings up a [PopupMenu] when clicked.
 	</brief_description>
 	<description>
-		Special button that brings up a [PopupMenu] when clicked. That's pretty much all it does, as it's just a helper class when building GUIs.
+		Special button that brings up a [PopupMenu] when clicked. 
+		New Items can be created inside this [PopupMenu] with code using [code]get_popup().add_item("My Item Name")[/code]. You can also create them directly from editor. First select MenueButton and then in the [code]Toolbar[/code], on top of editor, left click [code]Items[/code] and then left click [code]Add[/code]. You will be able to give each items new properties.
```
Comment (Calinou) [inline doc/classes/MenuButton.xml]:
```suggestion
		New items can be created inside this [PopupMenu] using [code]get_popup().add_item("My Item Name")[/code]. You can also create them directly from the editor. To do so, select the MenuButton node, then in the toolbar at the top of the 2D editor, click [b]Items[/b] then click [b]Add[/b] in the popup. You will be able to give each items new properties.
```

**Review by Calinou — CHANGES_REQUESTED** [review]

File: doc/classes/MenuButton.xml (hunk 1 above)
Comment (Feniks-Gaming) [reply doc/classes/MenuButton.xml]: done

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**